### PR TITLE
Fix for poo#18860

### PR DESCRIPTION
**Pull request by mimi1vx**
Start and enable serial-getty@hvc0 on SLE12  and SLE12SP1 if is
VIRTIO_CONSOLE set to 1

### CODE REVIEW

**Comment by mimi1vx** [comment]
http://argus.suse.cz/tests/779#step/shutdown/1 - SLE12 VIRTIO_CONSOLE=0
http://argus.suse.cz/tests/780#step/shutdown/4 - SLE12 VIRTO_CONSOLE=1
http://argus.suse.cz/tests/782#step/shutdown/1 - SLE12SP2

File: tests/shutdown/shutdown.pm [hunk 1]
```diff
@@ -14,20 +14,25 @@
...
+use utils 'sle_version_at_least';
 
 sub run {
 
     select_console('root-console');
     if (get_var('DROP_PERSISTENT_NET_RULES')) {
         type_string "rm -f /etc/udev/rules.d/70-persistent-net.rules\n";
     }
-
+    unless (sle_version_at_least('12-SP1') && !check_var('VIRTIO_CONSOLE', 1)) {
```
Comment (okurz) [inline tests/shutdown/shutdown.pm]: that `unless` paired with `!check_var` is really hard to follow. Can you use if?
